numpy/f2py/rules.py

```diff
@@ -206,6 +209,9 @@
 \t\t\"This module '#modulename#' is auto-generated with f2py (version:#f2py_version#).\\nFunctions:\\n\"\n#docs#\".\");
 \tPyDict_SetItemString(d, \"__doc__\", s);
 \tPy_DECREF(s);
+\ts = PyUnicode_FromString(""" + "\"" + numpy_version + "\");" + """
```

Comment:
This is a bit weird, as all the other macros use placeholders like `#docs#` etc. Can we use a `#numpy_version#` placeholder here too, and fill it where we fill the others?

Reply:
Ah nevermind, I see the generation time above is done the same way.